kafka-client/src/main/java/dev/responsive/kafka/api/stores/ResponsiveKeyValueBytesStoreSupplier.java

```diff
@@ -37,7 +37,7 @@ public String name() {
 
   @Override
   public KeyValueStore<Bytes, byte[]> get() {
-    return new ResponsiveStore(params);
+    return new ResponsiveKeyValueStoreWrapper(params);
```

Comment:
No change, just a renaming of `ResponsiveStore` to emphasize that it (a) is a KV store, and (b) is itself just a wrapper for the two KV sub-types, namely the global vs partitioned (regular) store